src/main/java/io/jenkins/plugins/services/impl/DefaultConfigurationService.java

```diff
@@ -107,6 +111,18 @@ public GeneratedPluginData getIndexData() throws ServiceException {
     }
   }
 
+  public String getJiraURL() {
+    return "https://issues.jenkins.io";
```

Comment:
url should be externalised, and this isn't the live url afaik, it's a half finished migration and issues.jenkins-ci.org is the actual domain

Reply:
How is it half finished? Looks like the domains point to the same data.

I debated the configurable domain thing. For one it'll eventually get pulled from update center, but what scenario would it need to be changed that can't have a redeploy? The wiki is hard coded all over the place

Reply:
the jira instance thinks it's live at issues.jenkins-ci.org afaik, you can't make any changes via the UI on the issues.jenkins.io domain, you just get errors